action.yml

```diff
@@ -83,15 +91,59 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Log the next step action
+    run: echo ▷ Figuring out the environment Python version...
+    shell: bash
+  - name: Compute job Python version
+    id: compute-job-python-version
+    run: |
+      # Compute job Python version
+      import sys
+
+      # Input from GHA
+      job_python_version = '${{ inputs.job-python-version }}'
+      old_python_version = '${{ inputs.python-version }}'
+      ansible_core_version = '${{ inputs.ansible-core-version }}'
+
+      # Handle case that deprecated Python version parameter is provided
+      if old_python_version:
+          # Check whether the default of job-python-version was changed
+          if job_python_version != 'auto':
+              print(
+                  'ERROR: `python-version` and `job-python-version` cannot'
+                  ' both be supplied. Use only `job-python-version` and not'
+                  ' its deprecated alias `python-version`.'
+              )
+              sys.exit(1)
```

Comment:
By the way, if you use `sys.exit(<string value>)`, it's equivalent to printing out that string and setting `1` as the return code.

Reply:
Fixed in e1fc0e9a5d4cd43e3fa84e76aedf7aab3b8e1abc.